composer.json

```
@@ -17,6 +17,7 @@
         "barryvdh/laravel-debugbar": "^3.3",
         "facade/ignition": "^1.16.4",
         "fakerphp/faker": "^1.9.1",
+        "laravel/ui": "^1.2",
```

[nits] module は `~` や　`^` は利用せず、バージョンは固定した方が良いと思ってます。

```suggestion
        "laravel/ui": "1.2",
```

また何かしらの理由で固定しない場合、以下のように記載した方が圧倒的にわかりやすいとも考えてます。

```suggestion
        "laravel/ui": ">=1.2 <2.0",
```